content/en/synthetics/ci.md

```diff
@@ -540,7 +540,7 @@ Variables to replace in the test. This object should contain, as keys, the name
 
 `pollingTimeout`
 : **Type**: integer<br>
-Duration after which `datadog-ci` should stop polling for test results (in milliseconds). Test results finishing after this duration are considered failed test results at the CI level.
+Duration after which `datadog-ci` should stop polling for test results (in milliseconds). Test results finishing after this duration are considered failed test results at the CI level. Defaults to 120000 ms.
```

Comment:
The duration in milliseconds after which `datadog-ci` stops polling for test results. The default is 120,000 ms. 

At the CI level, test results completed after this duration are considered failed.